File: programmers/[Lv.1]K번째 수/Solution.java
import java.util.ArrayList;
import java.util.Collections;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        for(int i=0;i<commands.length;i++) {
            ArrayList<Integer> list = new ArrayList();
            for(int j=commands[i][0];j<=commands[i][1];j++) {
                list.add(array[j-1]);
            }
            Collections.sort(list);
            answer[i] = list.get(commands[i][2]-1);
        }
        return answer;
    }
}

File: programmers/[Lv.1]K번째 수/Solution.swift
import Foundation

func solution(_ array:[Int], _ commands:[[Int]]) -> [Int] {
    var answer:[Int] = []
    for i in commands {
        var newArray = array[i[0]-1...i[1]-1]
        newArray.sort()
        print(array)
        //answer.append(newArray[i[2]-1])
    }
    return answer
}